.gitlab/build-oci-package.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -e
+
+mkdir sources
+
+cp -r ../tmp sources
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ Question:** ‏Shouldn't it be this instead?
```suggestion
cp -r ../tmp/* sources
```

Else the content of the package looks like this: 
```
$ ls -lah
total 16K
drwxr-xr-x 3 root root 4.0K May 27 11:31 .
drwxr-xr-x 3 root root 4.0K May 27 11:31 ..
drwxr-xr-x 6 root root 4.0K May 27 11:31 tmp
-rw-r--r-- 1 root root   41 May 27 11:31 version
```